src/modules/api/wallet/services/index.ts

```diff
@@ -1715,31 +1716,69 @@ export class WalletService {
         });
     }
 
-    async getTotalWalletBalance() {
-        const totalBalance = await this.prisma.wallet.aggregate({
+    private async aggregateTotalWalletBalance() {
```

Comment:
This should return an defined interface for commission balance and main balance. Also specify the return type in the method declaration